tests/acceptance/test_author_creation.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+from inspirehep.bat.pages import (
+    create_author,
+    holding_panel_author_detail,
+    holding_panel_author_list,
+)
+
+
+input_author_data = {
+    'given-names': 'Mark',
+    'family-name': 'Twain',
+    'display-name': 'M. Twain',
+    'native-name': 'M. Twain',
+    'public-email': 'mark.twain@history.org',
+    'orcid': '1111-1111-1111-1111',
+    'websites-0': 'http://www.example1.com',
+    'websites-1': 'http://www.example2.com',
+    'linkedin-url': 'http://www.example3.com',
+    'twitter-url': 'http://www.example4.com',
+    'blog-url': 'http://www.example5.com',
+    'institution-name': 'CERN',
+    'institution-start_year': '2000',
+    'institution-end_year': '2001',
+    'institution-rank': 'STAFF',
+    'experiments-name': 'ATLAS',
+    'experiments-start_year': '2002',
+    'experiments-end_year': '2005',
+    'advisors-name': 'Bob White',
+    'advisors-degree_type': 'PhD',
+    'comments': 'Some comments about the author',
+    'subject-0': 'ACC-PHYS',
+    'subject-1': 'ASTRO-PH'
+}
+
+
+def test_institutions_typehead(login):
+    create_author.go_to()
+    assert create_author.write_institution('cer', 'CERN').has_error()
```

Comment:
Guys we should move the assertion inside the lambda function/method that is executed in the has_error(). In that case the has_error() it would make more sense.

It can be refactorized in order to create a more readable code. ANyway I do not thing that is "badly designed"